### PR TITLE
[CI] running small tests only for RANS

### PR DESCRIPTION
**Pull request by philbucher**
**Description**
This is how it will be in the future (after I finally manage to add the mpi-test-run-script :)

For now only the rans tests are taking a lot of time hence I would go ahead with them

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -215,7 +215,7 @@ jobs:
         . /opt/intel/mkl/bin/mklvars.sh
         export PYTHONPATH=${PYTHONPATH}:${GITHUB_WORKSPACE}/bin/${{ matrix.build-type }}
         export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${GITHUB_WORKSPACE}/bin/${{ matrix.build-type }}/libs
-        mpiexec -np 2 python3 applications/RANSApplication/tests/test_RANSApplication_mpi.py --using-mpi
+        mpiexec -np 2 python3 applications/RANSApplication/tests/test_RANSApplication_mpi.py --using-mpi -l small
```
Comment (sunethwarna) [inline .github/workflows/ci.yml:218]: shouldn't this be
```suggestion
        mpiexec -np 2 python3 applications/RANSApplication/tests/test_RANSApplication_mpi.py --using-mpi -l mpi_small
```

File: .github/workflows/nightly_build.yml [hunk 2]
```diff
@@ -54,6 +54,14 @@ jobs:
         export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${GITHUB_WORKSPACE}/bin/${KRATOS_BUILD_TYPE}/libs
         python3 kratos/python_scripts/run_tests.py -l nightly -c python3
 
+    - name: Running MPI Python RANSApplication (2 Cores)
+      timeout-minutes : 10
+      run: |
+        . /opt/intel/mkl/bin/mklvars.sh
+        export PYTHONPATH=${PYTHONPATH}:${GITHUB_WORKSPACE}/bin/${{ matrix.build-type }}
+        export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${GITHUB_WORKSPACE}/bin/${{ matrix.build-type }}/libs
+        mpiexec -np 2 python3 applications/RANSApplication/tests/test_RANSApplication_mpi.py --using-mpi -l nightly
```
Comment (sunethwarna) [inline .github/workflows/nightly_build.yml:63]: This also?

```suggestion
        mpiexec -np 2 python3 applications/RANSApplication/tests/test_RANSApplication_mpi.py --using-mpi -l mpi_nightly
```

**Review by sunethwarna — COMMENTED** [review]
I will put some tests to `mpi_small` suite. At the moment that is totally empty.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (philbucher) [reply .github/workflows/ci.yml:218]: no, see https://github.com/KratosMultiphysics/Kratos/pull/4120

File: .github/workflows/nightly_build.yml (hunk 2 above)
Comment (philbucher) [reply .github/workflows/nightly_build.yml:63]: no, see https://github.com/KratosMultiphysics/Kratos/pull/4120

**Comment by sunethwarna** [comment]
Thanks for the help @philbucher. I think this is ready for review :)

**Review by sunethwarna — APPROVED** [review]
Sorry, I did not notice untill now that RANS MPI tests were taking this much time in the CI. Now it should be ok.

**Comment by philbucher** [comment]
for the record, I started working on the runscript for the mpi-python tests, but will take some more time to finish

**Pull request merged**